test/VsixTesting.Xunit.Tests/VsixTesting.Xunit.Tests.csproj

```diff
@@ -27,4 +27,13 @@
   <ItemGroup>
     <Reference Include="Microsoft.CSharp" />
   </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\VsixTesting.Invoker\VsixTesting.Invoker.csproj" ReferenceOutputAssembly="false" />
+    <ProjectReference Include="..\\..\src\VsixTesting.Invoker\VsixTesting.Invoker.csproj" ReferenceOutputAssembly="false" CopyVsix="true" ExtensionsDirectory="CopyVsix/True.Relative" />
```

Comment:
❓ If you build the test project with `CopyVsix="true"` for one of the references, and then change the value to `false` and build again (with an explicit Clean or Rebuild), is the previously-copied VSIX file removed?

Reply:
Yes. The `Rebuild` target depends on the `Clean` target so my previous answer also applies here.